scripts/python-skywater-pdk/skywater_pdk/generate_symbols.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 SkyWater PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+import sys
+import argparse
+from pathlib import Path
+import errno
+import contextlib
+import traceback
+import subprocess
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(prog=argv[0])
+    parser.add_argument(
+        'libraries_dir',
+        help='Path to the libraries directory of skywater-pdk',
+        type=Path
+    )
+    parser.add_argument(
+        'output_dir',
+        help='Path to the output directory',
+        type=Path
+    )
+    parser.add_argument(
+        '--libname',
+        help='Library name to generate the Symbolator diagrams for',
+        type=str
+    )
+    parser.add_argument(
+        '--version',
+        help='Version for which the Symbolator diagrams should be generated',
+        type=str
+    )
+    parser.add_argument(
+        '--create-dirs',
+        help='Create directories for output when not present',
+        action='store_true'
+    )
+    parser.add_argument(
```

Comment:
I don't get the `failed_inputs`? Is this an `allowed_to_fail`?

Reply:
This is a path to a file that has list of files for which the Symbolator failed to generate diagram

Reply:
Is this an output or input file?

Reply:
it is an output file

Reply:
I fixed the help for this argument